cypress.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
whoops. this file isn't needed at top-level any more. you can squash it into the last commit or something. was left behind when i relocated stuff.